omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -82,10 +82,14 @@ done
 
 if is_darwin; then
 
-  echo "This script is about to run another script."
-  sh /opt/chef-workstation/embedded/service/workstation-gui/poststeps.sh
-  echo "This script has just run another script."
-
+  # this will be removed when we bring back the workstation-gui changes back 
+  FILE="$HOME/Library/LaunchAgents/io.chef.chef-workstation.plist"
+  if [ -f "$FILE" ]; then
+    sudo -u $USER launchctl unload $HOME/Library/LaunchAgents/io.chef.chef-workstation.plist
+    sudo rm -rf ~/Library/LaunchAgents/io.chef.chef-workstation.plist
```

Comment:
better to use `$HOME/Library/...` here as well?

Reply:
Yeah right, I'll update it